plugins/woocommerce/includes/admin/class-wc-admin-assets.php

```diff
@@ -499,6 +499,11 @@ public function admin_scripts() {
 				);
 			}
 
+			// Email settings.
+			if ( $wc_screen_id . '_page_wc-settings' === $screen_id && isset( $_GET['tab'] ) && 'email' === $_GET['tab'] ) {
+				wp_enqueue_media();
+			}
+
```

Comment:
Is this also still needed? I tried removing it, and the media library still opened and worked.

Reply:
See my [other comment](https://github.com/woocommerce/woocommerce/pull/52615#discussion_r1840235959).

Reply:
Let's keep it. Just looking at the code, it makes sense to enqueue it, as you're introducing the media library modal. Maybe something specific to my testing why it worked even after removing.